sessionctx/variable/tidb_vars.go

```diff
@@ -155,6 +155,10 @@ const (
 
 // TiDB system variable names that both in session and global scope.
 const (
+
+	// tidb_opt_distinct_agg_push_down is used to decide whether agg with distinct should be pushed to tikv/tiflash.
+	TiDBOptDistinctAggPushDown = "tidb_distinct_agg_push_down"
```

Comment:
```suggestion
	TiDBOptDistinctAggPushDown = "tidb_opt_distinct_agg_push_down"
```

And please put it together with other `tidb_opt_xxx` options. So that we can show all the optimizer options with `show variables like "%tidb%opt%`;

Reply:
Addressed.